src/ch/imedias/rsccfx/view/RsccRequestPresenter.java

```diff
@@ -60,6 +83,26 @@ private void attachEvents() {
   }
 
   /**
+   *  Events for the supporter buttons, has to be called by createNewSupporterBtn too.
+   */
+  private void attachButtonEvents() {
+    for (int i = 0; i < buttons.size(); i++) {
```

Comment:
it seems that this is a bit hacky. I created a new issue, François might be able to stream it.